crates/sui-types/src/messages_grpc.rs

```diff
@@ -185,7 +170,7 @@ pub struct HandleCertificateResponseV2 {
 #[derive(Clone, Debug, Serialize, Deserialize)]
 pub struct SubmitCertificateResponse {
     /// If transaction is already executed, return same result as handle_certificate
-    pub executed: Option<HandleCertificateResponse>,
+    pub executed: Option<HandleCertificateResponseV2>,
```

Comment:
I believe this is safe to do because no one is calling `submit_certificate`. But flagging it just in case.

Reply:
agreed